authors/apps/authentication/views.py

```diff
@@ -72,3 +85,78 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ResetPasswordAPIView(generics.CreateAPIView):
+    """Sends password reset link to email"""
+    serializer_class = ResetSerializerEmail
```

Comment:
Please include `permission_classes` to make it explicit

Reply:
I don't think the route should be protected because you don't have to be logged in to access this route.

Reply:
Is (AllowAny,) not a permission class?